reporter/src/main/kotlin/reporters/TableReporter.kt

```diff
@@ -235,11 +235,11 @@ abstract class TableReporter : Reporter() {
             val tableRows = (listOf(project.id) + project.collectDependencyIds()).map { id ->
                 val scanResult = scanRecord.scanResults.find { it.id == id }
 
-                val scopes = project.scopes.filter { id in it }.associate {
-                    val scopeExcludes = ortResult.repository.config.excludes?.findScopeExcludes(it, project)
-                            ?: emptyList()
-                    Pair(it.name, scopeExcludes)
-                }.toSortedMap()
+                val scopes = project.scopes.filter { id in it }?.let { scopes ->
```

Comment:
This should still be a plain `.` instead of `?.`, or?

Reply:
Done.